tests/test_notify.py

```diff
@@ -0,0 +1,451 @@
+import datetime
+import subprocess
+from pathlib import Path
+
+PROJECT_PATH = Path(__file__).resolve().parents[1]
+DEFAULT_BUILDER_CONF = PROJECT_PATH / "tests/builder.yml"
+
+
+def get_issue(issue_title, repository):
+    issue = None
+    for i in repository.get_issues():
+        if i.title == issue_title:
+            issue = i
+            break
+    return issue
+
+
+def test_notify_00_template_build_success_upload(
+    token, github_repository, tmpdir_factory
+):
+    tmpdir = tmpdir_factory.mktemp("github-")
+    build_log = "dummy"
+    timestamp = datetime.datetime.utcnow().strftime("%Y%m%d%H%M%s")
+    template_name = "fedora-42"
+    package_name = f"qubes-template-{template_name}-4.2.0-{timestamp}"
+    distribution = "vm-fc42"
+
+    #
+    # build
+    #
+
+    status = "building"
+    cmd = [
+        str(PROJECT_PATH / "utils/notify-issues"),
+        f"--auth-token={token}",
```

Comment:
I don't like tokens on the command line, even in CI. Command line can be read by any process (including other users) on the same system. Use env variable as the actual builderv2-github does (in fact the env variable is already set in gitlab-ci). The same applies to other calls below.